templates/apiMultipartHelper.hbs

```diff
@@ -0,0 +1,18 @@
+import { Request, Response, NextFunction } from 'express'
+import multer from 'multer'
+import { Api } from '../../models'
+
+export const upload = multer({storage: multer.memoryStorage()})
+
+{{#each operations}}
+{{#if (hasOperationSupportingMultipart .)}}
+export namespace {{className name}} {
```

Comment:
Does this include the group name of the operation? Maybe we should have an outer namespace for the operation group name.